### PR TITLE
nxos_lacp_interfaces: Updating the tests to handle platforms that don't support - lacp mode delay

### PR DESCRIPTION
**Pull request by nkshrishail**
##### SUMMARY
The tests under nxos_lacp_interfaces were failing due to some image versions and some platforms not supporting the "lacp mode delay" command under port-channel interface configuration. Have updated the tests to handle such platforms.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
nxos_lacp_interfaces

### CODE REVIEW

File: test/integration/targets/nxos_lacp_interfaces/tests/cli/deleted.yaml [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 - set_fact:
     mode: delay
-  when: platform is not search('N5K|N6K|N7K') and ((imagetag != 'I2'))
+  when: platform is not search('N3K|N5K|N6K|N7K') and imagetag is not search('A8|I2') and
+    image_version is not search ('9.2') and chassis_type is not search('C9508')
```
Comment (mikewiebe) [inline test/integration/targets/nxos_lacp_interfaces/tests/cli/deleted.yaml]: Why did you need the `chassis_type` here?

Comment (nkshrishail) [reply test/integration/targets/nxos_lacp_interfaces/tests/cli/deleted.yaml]: Delayed LACP is not supported on Layer 3 port channels, FEX modules, or the Cisco Nexus 9500 Series switch. Platform for 9500 still set as N9K, I used chassis_type to actually check its a 9500 switch. And, I realized, I need to broaden the search to include other 9500 series switches C9504,C9508,C9516. 

I will do 
#######
chassis_type is not search('C95')
########

Comment (mikewiebe) [reply test/integration/targets/nxos_lacp_interfaces/tests/cli/deleted.yaml]: Gotcha.  Can you add a comment in the test to indicate this?

Comment (nkshrishail) [reply test/integration/targets/nxos_lacp_interfaces/tests/cli/deleted.yaml]: Done

**Comment by mikewiebe** [comment]
Changes LGTM 👍

**Review by mikewiebe — APPROVED** [review]

**Comment by chrisvanheuveln** [comment]
shipit

**Review by NilashishC — APPROVED** [review]
